src/Gateway.php

```diff
@@ -164,6 +164,54 @@ public function updateSubscription(array $parameters = array())
         return $this->createRequest('\Omnipay\Vindicia\Message\CreateSubscriptionRequest', $parameters, true);
     }
 
+    /**
+     * Fetch the invoice number of a subscription
+     *
+     * See Message\FetchSubscriptionInvoiceRequest for more details.
+     *
+     * @param array $parameters
+     * @return \Omnipay\Vindicia\Message\FetchSubscriptionInvoiceRequest
+     */
+    public function fetchSubscriptionInvoiceNumbers(array $parameters = array())
```

Comment:
i think fetchSubscriptionInvoiceReferences would be a better name. it is really weird that vindicia calls it an invoice number in some places and an invoice ID in others and we don't need to repeat that mistake